pkg/proxy/proxy.go

```diff
@@ -232,23 +304,26 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 				return
 			}
 			if p.isNotSupported(vin) {
-				p.forwardRequest(acct.Host, w, req)
+				p.forwardRequest(acct, w, req)
+				if acct.Host != p.fetchDomainForSubject(acct.Subject) {
+					p.updateDomainForSubject(acct.Subject, acct.Host)
```

Comment:
Redundant?

Reply:
You mean the acct.Host is already updated? this is to update the cache on the proxy level which can be used for followup proxy calls.